libraries/support/azure/rest.rb

```diff
@@ -5,6 +5,9 @@
 
 module Azure
   class Rest
+    USER_AGENT = 'User-Agent'
+    INSPEC_USER_AGENT = 'Chef-Inspec-18d63047-6cdf-4f34-beed-62f01fc73fc2'
```

Comment:
this must be:
INSPEC_USER_AGENT = 'pid-18d63047-6cdf-4f34-beed-62f01fc73fc2'
you can add others if needed

Reply:
@bmoore-msft "you can add others if needed" so we should generate GUIDs for our specific subproducts rather than tagging them in the agent?

Reply:
We need exactly one string in the "pid-GUID" format...  if there is other info you want to derive from the UA (not related to our tracking) then, you can still add it to the UA, but it will be ignored in the query.  That help?